python/gink/impl/server.py

```diff
@@ -45,12 +46,18 @@ def start_listening(self, addr="", port: Union[str, int] = "8080", auth: Optiona
         """ Listen for incoming connections on the given port.
         """
         port = int(port)
-        self._logger.info("starting to listen on %r:%r", addr, port)
-        listener = Listener(addr=addr, port=port, auth=auth)
+        certfile = environ.get("GINK_CERTFILE")
+        keyfile = environ.get("GINK_KEYFILE")
```

Comment:
It would be best to make `certfile` and `keyfile` arguments to `start_listening`.  We might want to have a secure listener on one port (thats facing the external world) and an insecure listener on another port (that's only listening inside of a nat).  Also, as a library, `Gink` shouldn't be looking at environment variables except in cases where it is using the library (e.g. __main__.py)